src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -320,6 +320,38 @@ public static boolean isOlderThanAYear(Date compareDate) {
         return (differenceInDays > 365);
     }
     
+    /**
+     * Checks if the time falls between the time range specified.
+     * Start time is inclusive (i.e. {@code startTime <= time < endTime})
+     * @param startTime the start time of the time range
+     * @param endTime the end time of the time range
+     * @param time the time to be checked
+     * @return true if the time falls between the start and end time
+     */
+    public static boolean isStartTimeWithinRange(Date startTime, Date endTime, Date time) {
+        if (startTime == null || endTime == null || time == null) {
+            return false;
+        }
+        
+        return startTime.getTime() <= time.getTime() && time.getTime() < endTime.getTime();
+    }
+    
+    /**
+     * Checks if the time falls between the time range specified.
+     * End time is inclusive (i.e. {@code startTime < time <= endTime})
+     * @param startTime the start time of the time range
+     * @param endTime the end time of the time range
+     * @param time the time to be checked
+     * @return true if the time falls between the start and end time
+     */
+    public static boolean isEndTimeWithinRange(Date startTime, Date endTime, Date time) {
+        if (startTime == null || endTime == null || time == null) {
+            return false;
+        }
+        
+        return startTime.getTime() < time.getTime() && time.getTime() <= endTime.getTime();
+    }
```

Comment:
These two methods do the same thing right? i.e. checking the first parameter is withing the 2nd and the 3rd. As they are now utility methods, they should be written in a more general way. Also check if a similar method exist already.